### PR TITLE
fix: 修复测试用例

### PR DESCRIPTION
**Pull request by JusterZhu**

### CODE REVIEW

File: test/Npgsql.PluginTests/GeoJSONTests.cs [hunk 1]
```diff
@@ -13,8 +13,8 @@
 using static Npgsql.Tests.TestUtil;
 
 namespace Npgsql.PluginTests;
-
```
Comment (WeihanLi) [inline test/Npgsql.PluginTests/GeoJSONTests.cs:16]: Maybe add the comment to the solution file and comment the plugin test project to avoid duplicates

File: src/Npgsql/Internal/NpgsqlDatabaseInfo.cs [hunk 2]
```diff
@@ -95,12 +95,12 @@ public abstract class NpgsqlDatabaseInfo
     /// <summary>
     /// Whether the backend supports the DISCARD TEMP statement.
     /// </summary>
-    public virtual bool SupportsDiscardTemp => Version.IsGreaterOrEqual(8, 3);
+    public virtual bool SupportsDiscardTemp => false; //Version.IsGreaterOrEqual(8, 3);
```
Comment (WeihanLi) [inline src/Npgsql/Internal/NpgsqlDatabaseInfo.cs:98]: Maybe we could remove the commented code to keep the code clean

**Comment by liubao68** [comment]
looks good to me

**Comment by JusterZhu** [comment]
> looks good to me

Done

**Review by WeihanLi — APPROVED** [review]

File: src/Npgsql/Internal/NpgsqlDatabaseInfo.cs [hunk 3]
```diff
@@ -92,15 +92,25 @@ public abstract class NpgsqlDatabaseInfo
     /// </summary>
     public virtual bool SupportsUnlisten => Version.IsGreaterOrEqual(6, 4);  // overridden by PostgresDatabase
 
+    #region Not supported DISCARD
+
+    /**
+     * todo:因为不支持DISCARD所以移除了
+     * Version.IsGreaterOrEqual(8, 3);
+     * 将所有设置为false。
+     */
```
Comment (liubao68) [inline src/Npgsql/Internal/NpgsqlDatabaseInfo.cs:101]: 我们可以不用考虑对于老版本兼容有关的代码。 只考虑目前使用的版本。 类似的判断逻辑和代码可以选择删除。
